{{cookiecutter.project_slug}}/.travis.yml

```diff
@@ -15,7 +16,7 @@ env:
 install: pip install -U tox
 
 # command to run tests, e.g. python setup.py test
-script: tox
+script: tox -e $TOXENV
```

Comment:
Otherwise, tox runs tests for every environment multiple times.  I checked it in my project; you can find it here https://travis-ci.org/likeon/python-headhunterru/builds/114120373.